docker-compose.yaml

```diff
@@ -53,6 +53,7 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
+        - ENCLAVE_TYPE=singleton
```

Comment:
Create docker-compose-singleton.yaml and follow the approach in https://github.com/hyperledger/avalon/commit/1ba5385f33fe510be942d1675392659fb1e4bb99